tools/pipeline_perf_test/load_generator/loadgen.py

```diff
@@ -2,13 +2,18 @@
 import grpc
 import time
 import argparse
+import threading
+import concurrent.futures
 from opentelemetry.proto.collector.logs.v1 import logs_service_pb2_grpc, logs_service_pb2
 from opentelemetry.proto.logs.v1 import logs_pb2
 from opentelemetry.proto.common.v1 import common_pb2
 
 def create_log_record():
+    # Using a hardcoded timestamp (May 14, 2025 12:00:00 UTC in nanoseconds)
```

Comment:
Just curious. I'm guessing, if you want to save time, you could you hardcode the entire `ExportRequest` object use the same request instead of creating a new one repeatedly?

Reply:
yes that is the ultimate goal.

Reply:
done